scripts/install_deepchem_conda.ps1

```diff
@@ -55,4 +55,4 @@ conda install -y -q -c deepchem -c rdkit -c conda-forge -c omnia `
     biopython `
     numpy
 
-pip install --pre -U $tensorflow tensorflow-probability
+pip install -U tensorflow tensorflow-probability
```

Comment:
We need the `$` there.  `$tensorflow` is a variable that could be set to either `tensorflow` or `tensorflow-gpu`.

Reply:
Thank you for reviewing!

In TF v2, I seem that we can use `pip install tensorflow` command for CPU and GPU.
https://www.tensorflow.org/install/gpu.

Is there some reasons that we should still install `tensorflow-gpu` package?